Makefile

```diff
@@ -11,10 +10,11 @@ distclean:
 	rm -rf build dist inx locales artifacts win mac *.spec *.tar.gz *.zip electron/node_modules electron/dist electron/build/mac electron/build/mac-arm64 electron/build/win-ia32-unpacked electron/build/linux-unpacked electron/build/linux-arm64-unpacked
 
 distlocal:
-	make distclean && make BUILD=local VERSION=local-build dist
-
+	@OS="$(uname)"
```

Comment:
I think this one isn't necessary and won't have any effect, because every line in a Makefile recipe runs in a separate shell.